### ISSUE TITLE
It seems that Ursine couldn't handle the flowchart of the mermaid.

### ISSUE THREAD

**Issue by Konfido** [aCluelessDanny/typora-theme-ursine#40]
Here is my test codes:
~~~
```mermaid
graph LR
%% LR: left2right, TD/TB: top2down, RL/BT
	subgraph 2
    a == d ==> c
    c --> d((Decision))
    end
	subgraph 1
	A[Hard edge] -->re(Round edge)
	A -.-a
	re -.->C{circle}
    C --- X[X]
    C -->|Two| B[B]
    C -.Three .-> D[C]
    end
    style a fill:#f9f,stroke:#333,stroke-width:4px
    style C fill:#ccf,stroke:#f66,stroke-width:2px,stroke-dasharray: 5, 5
```
~~~
Problems  found:
- the arrow of directed line is missing
- the color of font and background need to be rendered slightly. I test another theme, "Night", and here is the comparison of results.
![image](https://user-images.githubusercontent.com/25447610/68411802-1b379000-01c6-11ea-9c88-9c60d7cce805.png)
![image](https://user-images.githubusercontent.com/25447610/68411761-0824c000-01c6-11ea-87d6-52d47617da28.png)

Please help to solve this problem.

**Comment by aCluelessDanny** [comment]
Mm yeah, mermaid diagrams are a bit finicky for now, so they'll slowly get a bit of refinement overtime. I'll get working on fixing these ones when I can

**Comment by aCluelessDanny** [comment]
After playing around with it for a while, I think this is the best I can do. Found a way to style the arrows more accurately too.
<img width="629" alt="image" src="https://user-images.githubusercontent.com/21282515/68836513-297f2380-0680-11ea-8771-ff0dcd702b72.png">
<img width="607" alt="image" src="https://user-images.githubusercontent.com/21282515/68836529-313ec800-0680-11ea-868f-5e5582e130a5.png">
As for the text color for "a" & "circle" in Umbra, I noticed you styled them with custom fill colors in the diagram code. Since the theme has to accommodate for both black and white text colors, it might be a bit tricky using a set node fill color if you switch between themes.
Not sure if the default node colors have any significance, but hopefully it looks alright now
